src/Server/modules/WorldPlaceManager.js

```diff
@@ -0,0 +1,245 @@
+			if (country == allWorldPlace[i].country && province == allWorldPlace[i].state) {
+				for (var key in allWorldPlace[i].numDeaths) {
+					var temp = new Date(key);
+					var d =
+						temp.getMonth() +
+						1 +
+						'/' +
+						temp.getDate() +
+						'/' +
+						temp.getFullYear();
+					if (d == date) {
+						allWorldPlace[i].numDeaths[key] = number;
+						result = true;
+						break;
+					}
+				}
+			}
+		}
+	} else if (tod == 3) {
+		console.log(allWorldPlace.length);
+		for (var i = 0; i < allWorldPlace.length; ++i) {
+			if (country == allWorldPlace[i].country && province == allWorldPlace[i].state) {
+				for (var key in allWorldPlace[i].numRecovered) {
+					var temp = new Date(key);
+					var d =
+						temp.getMonth() +
+						1 +
+						'/' +
+						temp.getDate() +
+						'/' +
+						temp.getFullYear();
+					if (d == date) {
+						allWorldPlace[i].numRecovered[key] = number;
+						result = true;
+						break;
+					}
+				}
+			}
+		}
+	}
+	return result;
+}
+
+function RemoveallWorldPlace(province, country, date, tod) {
+    if (allWorldPlace == undefined) InitializeAllWorldPlace();
+
+    var result = false;
+	if (tod == 1) {
+		for (var i = 0; i < allWorldPlace.length; ++i) {
+			if (country == allWorldPlace[i].country && province == allWorldPlace[i].state) {
+				for (var key in allWorldPlace[i].numConfirmed) {
+					var temp = new Date(key);
+					var d =
+						temp.getMonth() +
+						1 +
+						'/' +
+						temp.getDate() +
+						'/' +
+						temp.getFullYear();
+					if (d == date) {
+						delete allWorldPlace[i].numConfirmed[key];
+						console.log('DELETED');
+						result = true;
+						break;
+					}
+				}
+			}
+		}
+	} else if (tod == 2) {
+		for (var i = 0; i < allWorldPlace.length; ++i) {
+			if (country == allWorldPlace[i].country && province == allWorldPlace[i].state) {
+				for (var key in allWorldPlace[i].numDeaths) {
+					var temp = new Date(key);
+					var d =
+						temp.getMonth() +
+						1 +
+						'/' +
+						temp.getDate() +
+						'/' +
+						temp.getFullYear();
+					if (d == date) {
+						delete allWorldPlace[i].numDeaths[key];
+						console.log('DELETED DEATH');
+						result = true;
+						break;
+					}
+				}
+			}
+		}
+	} else if (tod == 3) {
+		for (var i = 0; i < allWorldPlace.length; ++i) {
+			if (country == allWorldPlace[i].country && province == allWorldPlace[i].state) {
+				for (var key in allWorldPlace[i].numRecovered) {
+					var temp = new Date(key);
+					var d =
+						temp.getMonth() +
+						1 +
+						'/' +
+						temp.getDate() +
+						'/' +
+						temp.getFullYear();
+					if (d == date) {
+						delete allWorldPlace[i].numRecovered[key];
+						console.log('DELETED RECOVERED');
+						result = true;
+						break;
+					}
+				}
+			}
+		}
+	}
+	return result;
+}
+
+function SaveRecords() {
+    if(allWorldPlaces == undefined) InitializeAllWorldPlaces();
+    writeCSVModule.RecordWorldData(allWorldPlace);
+}
```

Comment:
At very end of the file, we need export = { ... }. Look at CaseManager for template :D